src/Entity/User.php

```diff
@@ -183,7 +183,7 @@ public function getAvailableAssignmentById(int $assignmentId): Assignment
     {
         $assignment = $this->getAssignmentById($assignmentId);
 
-        if ($assignment->isAvailable()) {
+        if ($assignment->getLineItem()->isActive() && $assignment->isAvailable()) {
```

Comment:
I was thinking exactly the opposite. I prefer to expose it here. Any output of the other guys? @luisfn

Reply:
Maybe, but in that case naming of `isAvailable` method on Assignment entity doesn't make sense: it checks if the assignment is available, BUT, if it's not active, it cannot be available. So, IMO, `isActive` should be a part of `isAvailable` check.
In any case, indeed, it would be good to see @luisfn and @Skyld opinions